.github/workflows/cabal.project.yaml

```diff
@@ -20,12 +20,14 @@ jobs:
       fail-fast: false
       matrix:
         os: [macOS-12, windows-latest, ubuntu-latest]
-        ghc: ["8.10.7", "9.0.2", "9.2.8", "9.4.8"]
+        ghc: ["8.10.7", "9.0.2", "9.2.8", "9.4.8", "9.6.5"]
```

Comment:
Why not 9.6.6?

Reply:
Old branch was on 9.6.5, wasn't sure if you'd appreciate me bumping it.